client/modules/IDE/pages/IDEView.jsx

```diff
@@ -357,6 +357,7 @@ class IDEView extends React.Component {
                   provideController={(ctl) => { this.cmController = ctl; }}
                 />
                 <Console
+                  consoleFontSize={this.props.preferences.fontSize}
```

Comment:
given that the editor font size and the console font size are the same and based on the same `prop`, i think it's slightly confusing here to call this `prop` `consoleFontSize` and it should just be  `fontSize`.